solana.js

```diff
@@ -96,6 +98,47 @@ async function createWithSeedTx(authorityPublicKey, lamports, source) {
     return [createStakeAccountTx, stakeAccountPubkey, []]
 }
 
+async function split(authorityPublicKey, lamports, oldStakeAccountPubkey, source) {
+    // TODO add support of default split
+    // StakeProgram.split({
+    //     stakePubkey: stakeAccountPublicKey,
+    //     authorizedPubkey: senderPublicKey,
+    //     votePubkey: validatorPubkey,
+    // });
+        
+    // Format source to
+    seed = formatSource(source);
+   
+    const newStakeAccountPubkey = await PublicKey.createWithSeed(
+        authorityPublicKey,
+        seed,
+        StakeProgram.programId,
+    );
+
+    const splitStakeAccountTx = new Transaction().add(
+        StakeProgram.splitWithSeed({
+            stakePubkey: oldStakeAccountPubkey,
+            authorizedPubkey: authorityPublicKey,
+            splitStakePubkey: newStakeAccountPubkey,
+            basePubkey: authorityPublicKey,
+            seed: seed,
+            lamports: lamports
+        })
+       );
+        
+    return [splitStakeAccountTx, newStakeAccountPubkey, []]
```

Comment:
why we need 3rd value in this tuple ?

Reply:
Transaction could have additional signers